### ISSUE TITLE
(Voicing & PDOM) Go beyond-edge-responses - need some clarity

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#291]
In Interactive Description when a user presses an Arrow key when they are at the max or min position, they receive a response that describes the current state information rather than a resulting change.

When moving the mass spheres this means:
- qualitative distance, and qualitative and quantitative force information
    - Far from mass 2, force arrows tiny, forces 10.9 newtons. 
- just qualitative distance, and qualitative size of arrows when Force Values checkbox is not checked
    - Far from mass 2, force arrows tiny. 

When changing mass of a sphere:
- Much much smaller than mass 2, force arrows tiny, forces 5.4 newtons.
- Much much smaller than mass 2, force arrows tiny.

We currently have issues in both the Interactive Description design (PDOM) and the Voicing.

**Voicing Issue**
**When moving a sphere:**
- I hear an object and context responses describing force changes when trying to move beyond the edge min and max position. I think I should we should hear nothing in this case (will verify at next design meeting).
- Note when changing mass, I don't hear any go-beyond-edge responses when trying to go beyond max or min mass (this should be good as is).

**PDOM Issues**
**When moving mass spheres:**
- I get the correct responses, and
- I get them every time I try to go beyond the edge with an arrow-key press

**When changing mass:**
- I get the correct qualitative responses for mass, but
- I get quantitative force info with or without force values checked. Quantitative force should not be there when Force Values is not checked.
- I get these responses every time try to go beyond the edge with an arrow-key press

Edited issue July 27, correcting grammar and clarifying language.

**Comment by terracoda** [comment]
Questions for design meeting:
1. Do we all agree, we do not need go-beyond-edge responses in the Voicing feature for this sim?

**Comment by terracoda** [comment]
We discussed the go-beyond-edge responses at yesterday's Voicing Design meeting. Here are the decision to the questions above:
1. Voicing: Do we need go-beyond-edge responses in the Voicing feature for this sim? Decided we do not need go-beyond-edge responses for the Voicing feature in GFLB. It was also noted that these responses are somewhat bespoke - part object state information/part context state information. @terracoda to make an issue to keep the distance edge response from repeating.
2. PDOM: Can we remove quantitative force value from the mass change response when Force Values are not checked? Should not be a problem. @terracoda to make an issue.
3. PDOM: Is it possible to only deliver the go-beyond-edge response once? - Would need investigating. Not worth investigating now.
4. PDOM: (Eventually - maybe not now) Is it worth updating the common code for the accessible slider design pattern to allow for an option to have go-beyond-edge responses?  Not all sliders need them, but I have a feeling they are nice touch for some sliders and a nice touch for non-visual access. Yes, @terracoda to make a design issue that can gather more examples of where we have used go-beyond-edge responses.

**Comment by terracoda** [comment]
@terracoda will close this issue soon, once she makes sure all long-term issues have been created. Short-term issues already created.

**Comment by terracoda** [comment]
#292 and #293 have been verified, so closing.
